inc/saihostintf.h

```diff
@@ -212,101 +222,121 @@ typedef enum _sai_hostif_trap_type_t
     /** default packet action is forward */
     SAI_HOSTIF_TRAP_TYPE_VRRP = 0x00002005,
 
-    /**
-     * @brief BGP traffic (tcp src port == 179 or tcp dst port == 179) to local
-     * router IP address (default packet action is forward)
-     */
-    SAI_HOSTIF_TRAP_TYPE_BGP = 0x00002006,
-
     /** default packet action is forward */
-    SAI_HOSTIF_TRAP_TYPE_DHCPV6 = 0x00002007,
+    SAI_HOSTIF_TRAP_TYPE_DHCPV6 = 0x00002006,
 
     /** default packet action is forward */
-    SAI_HOSTIF_TRAP_TYPE_OSPFV6 = 0x00002008,
+    SAI_HOSTIF_TRAP_TYPE_OSPFV6 = 0x00002007,
 
     /** default packet action is forward */
-    SAI_HOSTIF_TRAP_TYPE_VRRPV6 = 0x00002009,
-
-    /**
-     * @brief BGPv6 traffic (tcp src port == 179 or tcp dst port == 179) to
-     * local router IP address (default packet action is forward)
-     */
-    SAI_HOSTIF_TRAP_TYPE_BGPV6 = 0x0000200a,
+    SAI_HOSTIF_TRAP_TYPE_VRRPV6 = 0x00002008,
 
     /** default packet action is forward */
-    SAI_HOSTIF_TRAP_TYPE_IPV6_NEIGHBOR_DISCOVERY = 0x0000200b,
+    SAI_HOSTIF_TRAP_TYPE_IPV6_NEIGHBOR_DISCOVERY = 0x00002009,
 
     /** default packet action is forward */
-    SAI_HOSTIF_TRAP_TYPE_IPV6_MLD_V1_V2 = 0x0000200c,
+    SAI_HOSTIF_TRAP_TYPE_IPV6_MLD_V1_V2 = 0x0000200a,
 
     /** default packet action is forward */
-    SAI_HOSTIF_TRAP_TYPE_IPV6_MLD_V1_REPORT = 0x0000200d,
+    SAI_HOSTIF_TRAP_TYPE_IPV6_MLD_V1_REPORT = 0x0000200b,
 
     /** default packet action is forward */
-    SAI_HOSTIF_TRAP_TYPE_IPV6_MLD_V1_DONE = 0x0000200e,
+    SAI_HOSTIF_TRAP_TYPE_IPV6_MLD_V1_DONE = 0x0000200c,
 
     /** default packet action is forward */
-    SAI_HOSTIF_TRAP_TYPE_MLD_V2_REPORT = 0x0000200f,
+    SAI_HOSTIF_TRAP_TYPE_MLD_V2_REPORT = 0x0000200d,
+
+    /** Router traps custom range start */
+    SAI_HOSTIF_TRAP_TYPE_ROUTER_CUSTOM_RANGE_BASE = 0x0003000,
+
+    /* local IP traps */
 
     /**
      * @brief IP packets to local router IP address (routes with
      * #SAI_ROUTE_ENTRY_ATTR_NEXT_HOP_ID = #SAI_SWITCH_ATTR_CPU_PORT)
-     * (default packet action is trap)
+     * (default packet action is drop)
```

Comment:
Correct. SSH, SNMP, BGP are more specific cases of IP2ME with higher prio
Please also note this hasn't changed at all from the previous spec version definition